packages/sdk/src/mnemonic.ts

```diff
@@ -54,15 +56,18 @@ export class Mnemonic {
   /**
    * Validate a mnemonic string, raise an exception providing reason
```

Comment:
I think this comment is outdated now that the return type has changed.

Reply:
Yeah we should avoid mentioning any type information in the description :D